packages/monitoring/src.ts/reserve-data.ts

```diff
@@ -0,0 +1,164 @@
+import {
+  getAllMarketsData,
+  AssetSummary,
+  PRICING_DECIMALS,
+} from "@vmexfinance/sdk";
+import dotenv from "dotenv";
+dotenv.config();
+
+export function getProviderRpcUrl(network: string): string {
+  if (network == "sepolia") {
+    return "https://eth-sepolia.public.blastapi.io";
+  } else if (network == "optimism") {
+    return `https://optimism.llamarpc.com/rpc/${process.env.REACT_APP_LLAMA_RPC_KEY}`;
+  } else if (network == "base") {
+    return `https://base-mainnet.public.blastapi.io`;
+  } else if (network == "arbitrum") {
+    return `https://arbitrum-one.public.blastapi.io`;
+  }
+
+  return "";
+}
+
+const vmexAlertsDiscordWebhook = process.env.DISCORD_ALERTS_WEBHOOK_URL;
+
+export function sendAlert(
+  message: string,
+  severity: number,
+  messages: string[]
+) {
+  let alertMessage = `SEV ${severity} ALERT: ${message}`;
+  if (severity <= 2) {
+    alertMessage += "\n@everyone";
+  }
+  console.error(alertMessage);
+  messages.push(alertMessage);
+  // TODO: More alerting depending on severity
+}
+
+function sendMessage(message: string) {
+  if (!vmexAlertsDiscordWebhook) {
+    throw new Error(
+      "Vmex discord messages webhook is not defined in env variables"
+    );
+  }
+  const data = typeof message === "string" ? { content: message } : message;
+  return new Promise<void>((resolve, reject) => {
+    fetch(vmexAlertsDiscordWebhook, {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify(data),
+    })
+      .then((response) => {
+        if (!response.ok) {
+          reject(new Error(`Could not send message: ${response.status}`));
+        }
+        resolve();
+      })
+      .catch((error) => {
+        console.error(error);
+        reject(error);
+      });
+  });
+}
+
+function checkPermanentLoss(
```

Comment:
I would rename this to `checkMarketLiquidatable` or `checkMarketLockedExceedsDebt`